### PR TITLE
Add top level module list

### PR DESCRIPTION
**Pull request by m6jones**
Made a module for top level module list that will be used in learning paths:

Collapsed:
image

Expanded:
image

Design:
image

image

Completion is not in scope.

### CODE REVIEW

File: lang/en.json [hunk 1]
```diff
@@ -1,25 +1,28 @@
 {
```
Comment (m6jones) [inline lang/en.json]: Sorry for the mess, I sorted the list

File: lang/en.json [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
+	"gotoOnedrive": "Go to OneDrive",
+	"hideModules": "Hide {numberOfModules} Modules",
 	"imDone": "I'm Done",
-	"niceWork": "Nice work so far!",
+	"invalidFile": "This kind of file can't be opened in this viewer",
+	"iterateToNext": "Next activity",
+	"iterateToParent": "Go to parent activity",
+	"iterateToPrevious": "Previous activity",
 	"missedActivities": "You <d2l-link href=\"javascript:void(0)\">missed {count, plural, one {# activity} other {# activities}}</d2l-link>.",
+	"module": "Module",
```
Comment (m6jones) [inline lang/en.json]: This one is new

File: lang/en.json [hunk 3]
```diff
@@ -1,25 +1,28 @@
...
+	"niceWork": "Nice work so far!",
 	"noNeedToFinish": "If you don't need to finish {count, plural, one {it} other {them}}, no problem.",
-	"showMissed": "Show Me What I Missed",
-	"youMissedThese": "You missed these activities:",
+	"openNew": "Open in a new window",
 	"openOnedriveFile": "Open OneDrive file in a new browser tab.",
-	"gotoOnedrive": "Go to OneDrive"
+	"openUnsecureContent": "Your browser thinks this content is published in a less secure location. Don't worry, just open the content in a new window.",
+	"showMissed": "Show Me What I Missed",
+	"showModules": "Show {numberOfModules} Modules",
```
Comment (m6jones) [inline lang/en.json]: This one is new

File: lang/en.json [hunk 4]
```diff
@@ -1,25 +1,28 @@
...
-	"undisplayableContent": "This content can't be displayed in this viewer",
-	"openUnsecureContent": "Your browser thinks this content is published in a less secure location. Don't worry, just open the content in a new window.",
-	"viewThisActivity": "View this Activity.",
 	"goToActivity": "Go to Activity",
-	"openNew": "Open in a new window",
-	"cannotBeRendered": "This content item cannot be rendered.",
-	"congratulations": "Congratulations!",
-	"activitiesFinishedGreatJob": "You have completed all of the activities in this unit. Great job!",
+	"gotoOnedrive": "Go to OneDrive",
+	"hideModules": "Hide {numberOfModules} Modules",
```
Comment (m6jones) [inline lang/en.json]: this one is new

File: components/d2l-sequences-module-name.js [hunk 5]
```diff
@@ -26,7 +26,7 @@ export class D2LSequencesModuleName extends mixinBehaviors([
 	static get properties() {
 		return {
 			title: {
-				type: Object,
+				type: String,
```
Comment (benyamin-noori) [inline components/d2l-sequences-module-name.js]: Not sure why we had this as object.

File: components/d2l-sequences-module-list.js [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+					color: var(--d2l-color-tungsten);
+					display: block;
+					letter-spacing: 0.4px;
+					line-height: 1.19;
+					padding: 13px 20px;
+				}
+
+				.d2l-sequences-module-list-list li d2l-link:hover d2l-sequences-module-name {
+					background-color: var(--d2l-color-regolith);
+					border-bottom: 1px solid var(--d2l-color-mica);;
```
Comment (AdmiralCoco) [inline components/d2l-sequences-module-list.js]: Some extra semicolons here

File: demo/d2l-sequences-module-list.html [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+				<custom-style>
+					<style is="custom-style" include="demo-pages-shared-styles"></style>
+				</custom-style>
+				<custom-style include="d2l-typography">
+					<style is="custom-style" include="d2l-typography">
+						html {
+							font-size: 20px;
+						}
+						.fixed-size {
+							display:flex;
```
Comment (AdmiralCoco) [inline demo/d2l-sequences-module-list.html]: Nit: alphabetize

Comment (m6jones) [reply demo/d2l-sequences-module-list.html]: Even after I installed the program! Thanks :)

**Pull request merged**